SConstruct

```diff
@@ -1466,6 +1466,18 @@ else :
 	if not haveInkscape :
 		sys.stderr.write( "WARNING : Inkscape not found - not building docs. Check INKSCAPE build variable.\n" )
 
+#########################################################################################################
+# Documentation reference scripts
+#########################################################################################################
+
+referenceFiles = []
+for ext in ( 'grf', 'png' ) :
+	referenceFiles += glob.glob( "doc/references/*.%s" % ext )
+
+for f in referenceFiles :
+	fileInstall = env.Command( f.replace( "doc/", "$BUILD_DIR/resources/", 1 ), f, Copy( "$TARGET", "$SOURCE" ) )
```

Comment:
I wasn't sure if this was the best spot for reference scripts. I figure they'd be useful as further examples and as nodes for non-docs purposes, too.

Reply:
941996f7afa2e5376d7cd0c47265c64c9548c3cb